webpack/loaders/timeline-loader.cjs

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function(content, map, meta) {
+  let ret = '';
+
+  content.split(/\r?\n/).forEach((_line) => {
+    let line = _line.trim();
+    if (!line || line.startsWith('#'))
+      return;
+    line = line.replace(/#.*$/, '').trim();
+    ret += line + '\r\n';
```

Comment:
What about removing lines that are only comments but `#` doesn't start at the beginning of the line too?

```suggestion
    let line = _line.replace(/#.*$/, '').trim();
    if (!line)
      return;
    ret += line + '\r\n';
```

But more importantly, either of these will break `100.0 "Some ability #1" sync /stuff/`.  Can you fix the regex to handle # inside of quotes as well?

Reply:
Oh, yes.
I gonna fix it soon.

Reply:
How about this regex?
https://regex101.com/r/CkKpW6/1

The only issue is when comment contain one quote (") would causing regex unmatched.

Reply:
https://regex101.com/r/FwTpMe/2 maybe?

Reply:
lookbehind sounds good, but is it possible that '#' in `sync /regex/` things?

Reply:
Oh, that's quite right!

Reply:
Looks like javascript regex doesn't offer something like [`(?(exp)yes|no)`](https://docs.microsoft.com/en-gb/dotnet/standard/base-types/regular-expression-language-quick-reference?redirectedfrom=MSDN#alternation-constructs), so don't know how to capture the `sync /regex/` part as well.

Reply:
Hmm, how about: https://regex101.com/r/FwTpMe/3

Reply:
I tried several times and it works well, thanks!

Reply:
Wait sorry: https://regex101.com/r/FwTpMe/4

One character change to support slashes in the ability name!

Reply:
Haha, I see that you literally just updated this as I had this thought.